build-push-container-image/action.yml

```diff
@@ -114,7 +114,7 @@ runs:
       run: >
         dotnet publish --os linux --arch x64
         -p:PublishProfile=DefaultContainer
-        -p:ContainerImageName=${{inputs.imageName}}
+        -p:ContainerRepository=${{inputs.imageName}}
```

Comment:
I thought I had fixed this already in PR #67 , somehow it returned. All good, thanks!